journal-of-international-relations-and-development.csl

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Journal of International Relations and Development</title>
+    <title-short>JIRD</title-short>
+    <id>http://www.zotero.org/styles/journal-of-international-relations-and-development</id>
+    <link href="http://www.zotero.org/styles/journal-of-international-relations-and-development" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-elections-public-opinion-and-parties" rel="template"/>
+    <link href="http://www.tandf.co.uk/journals/authors/style/reference/ref_fwep.pdf" rel="documentation"/>
+    <author>
+      <name>Michal Mochtak</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issn>1408-6980</issn>
+    <eissn>1581-1980</eissn>
+    <updated>2017-04-22T19:54:57+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" initialize="false"/>
+      <label form="short" prefix=", " suffix=","/>
```

Comment:
going by the styleguide, this is in parentheses, i.e. (eds.) not , eds. -- adjust unless they actually do this differently in published articles.